sycl/include/sycl/ext/oneapi/experimental/fixed_size_group.hpp

```diff
@@ -12,6 +12,7 @@
 #include <sycl/detail/pi.h>            // for PI_ERROR_INVALID_DEVICE
 #include <sycl/detail/type_traits.hpp> // for is_fixed_size_group, is_group
 #include <sycl/exception.hpp>          // for runtime_error
+#include <sycl/ext/oneapi/experimental/non_uniform_groups.hpp>
```

Comment:
Nit; Should this have a comment about what is used from the header file?

Reply:
To be honest I'd drop all such comments. It's impossible to update all uses of a header when it stops including something.